src/registrar/views/application.py

```diff
@@ -319,13 +321,26 @@ def post(self, request, *args, **kwargs):
             self.save(forms)
         else:
             # unless there are errors
+            # no sec because this use of mark_safe does not introduce a cross-site
+            # scripting vulnerability because there is no untrusted content inside.
+            # It is only being used to pass a specific HTML entity into a template.
+            messages.warning(
+                request,
+                mark_safe(  # nosec
+                    "<b>We could not save all the fields.</b><br/> The highlight "
+                    + "fields below <b>could not be saved</b> because they have "
+                    + "missingor invalid data. All other information on this page "
+                    + "has been saved."
+                ),
+            )
             context = self.get_context_data()
             context["forms"] = forms
             return render(request, self.template_name, context)
 
         # if user opted to save their progress,
         # return them to the page they were already on
         if button == "save":
+            messages.success(request, "Your progress has been saved!")
```

Comment:
**comment**: This is so nice to see after clicking Save!

<img width="662" alt="Screen Shot 2023-02-17 at 12 13 20" src="https://user-images.githubusercontent.com/443389/219745087-d90a567c-6490-4503-8c07-4b8f798b63e9.png">